### PR TITLE
Update part endpoints

### PR DESCRIPTION
**Pull request by tsublette**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -2159,6 +2192,45 @@ definitions:
...
+    properties:
+      signedURL:
+        type: string
+
+  Triangle:
+    properties:
+      normal:
+        type: object
+        properties:
+          angle:
```
Comment (chadqueen) [inline swagger.yaml]: where does an angle property come from for a triangle? i do not see it on the model in untility

File: swagger.yaml [hunk 2]
```diff
@@ -2159,6 +2192,45 @@ definitions:
...
+          y:
+            type: number
+            format: double
+          z:
+            type: number
+            format: double
+      vertices:
+        type: array
+        items:
+          $ref: '#/definitions/Vertice'
```
Comment (chadqueen) [inline swagger.yaml]: /Vertex

File: swagger.yaml [hunk 3]
```diff
@@ -2159,6 +2192,45 @@ definitions:
...
+
+  Vertice:
```
Comment (chadqueen) [inline swagger.yaml]: Vertex

File: swagger.yaml (hunk 1 above)
Comment (tsublette) [inline swagger.yaml]: I don't know. It was in the response from the current API. We could take it out of the spec and make sure it is not included in the json serialization - no strong preference

**Pull request merged**